VideoLocker/src/main/java/org/edx/mobile/util/FileUtil.java

```diff
@@ -38,4 +41,25 @@
             inputStream.close();
         }
     }
+
+    /**
+     * Deletes a single file and/or recursively deletes files/folders within a folder before
+     * deleting it.
+     *
+     * @param fileOrDirectory The file/folder that needs to be deleted.
+     * @param exceptions      Name of the file/folder that needs to be skipped while deletion.
```

Comment:
The '/' here should be replaced with 'or'.

Also note that 'directory' is a technically more accurate terminology than 'folder'.